src/bin/solang.rs

```diff
@@ -177,6 +177,12 @@ fn main() {
                             .short('g')
                             .long("generate-debug-info")
                             .hide(true),
+                    )
+                    .arg(
+                        Arg::new("LOGAPIRETURNS")
+                            .help("Log the return codes of runtime API calls in the environment")
+                            .long("log-api-calls")
+                            .action(ArgAction::SetFalse),
```

Comment:
This is not right. The options above like `--no-cse` disable CSE, so those have `.action(ArgAction::SetFalse)`. In this case, want to enable something, so it should be `.action(ArgAction::SetTrue)`.

Actually it might be best to just remove the entire `.action(..)` line